src/NServiceBus.AcceptanceTests/Routing/When_subscribing_to_a_base_event_on_unicast_transports.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_subscribing_to_a_base_event_on_unicast_transports : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Specific_event_should_be_delivered()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b => b.When(c => c.SubscriberSubscribed, async session => await session.Publish(new SpecificEvent())))
+                .WithEndpoint<GeneralSubscriber>(b => b.When(async (session, c) => await session.Subscribe<IBaseEvent>()))
+                .Done(c => c.SubscriberGotEvent)
+                .Run();
+
+            Assert.True(context.SubscriberGotEvent);
+        }
+
+        [Test]
+        public async Task Base_event_should_only_be_delivered()
```

Comment:
Nit pick: We should only have a single test per class since it causes race conditions on some transports since the queue names will be the same so tests can interfere with each other